hack/conformance.sh

```diff
@@ -1,8 +1,10 @@
-set -xe
+#!/bin/bash
+
+set -xeuo pipefail
 
 ARTIFACTS=${ARTIFACTS:-${PWD}}
 sonobuoy_version=0.18.2
-config_file=$1
+config_file=${1:-}
 
 if [[ -z "$KUBEVIRT_PROVIDER" ]]; then
     echo "KUBEVIRT_PROVIDER is not set" 1>&2
```

Comment:
By specifying `1>&2` we would get the value of `KUBEVIRT_PROVIDER`?

Reply:
i didnt touch this line, out of this PR scope, 
anyhow its for redirecting stdout to stderr